### PR TITLE
27 generate api documentation

### PR DESCRIPTION
**Pull request by ylussaud**

### CODE REVIEW

File: doc/org.eclipse.python4capella.doc/doc/developer/DeveloperManual.textile [hunk 1]
```diff
@@ -9,3 +9,5 @@ h2. Advanced Python users
 h2. Python4Capella contributors
 
 * "*Create a Python interpreter distribution for Windows*":python/CreateWindowsPythonFile.html Describes how you can recreate the Python interpreter zip file for Windows.
+* "*Generate Python API*":python/GeneratePythonTests.html Describes how you can Python API for the Python Capella light metamodel.
```
Comment (mbats) [inline doc/org.eclipse.python4capella.doc/doc/developer/DeveloperManual.textile:12]: Verb is missing

File: doc/org.eclipse.python4capella.doc/doc/developer/python/GeneratePythonAPI.textile [hunk 2]
```diff
@@ -0,0 +1,19 @@
+h1. Generate Python API
+
+This section describes how you can generate Python API for the Capella light metamodel.
+
+The generator is defined in the "ProduceCapellaPythonAPIFromCapellaHandler":https://github.com/labs4capella/python4capella/blob/master/plugins/org.eclipse.python4capella.gen/src/org/eclipse/python4capella/gen/handlers/ProduceCapellaPythonAPIFromCapellaHandler.java Java class. It use the simplified Capella metamodel defined in the "specification folder":https://github.com/labs4capella/python4capella/tree/master/specification of the git repository.
```
Comment (mbats) [inline doc/org.eclipse.python4capella.doc/doc/developer/python/GeneratePythonAPI.textile:5]: uses

File: doc/org.eclipse.python4capella.doc/doc/developer/python/GeneratePythonAPI.textile [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+The Capella Light Metamodel is defined using Capella 5.0, you will need this version to generate the Python API.
+
+Start to import the "org.eclipse.python4capella.gen":https://github.com/labs4capella/python4capella/tree/master/plugins/org.eclipse.python4capella.gen project in your Capella 5.0 workspace and "enable developer capabilities":../../user/general/EnableDevelopmentCapabilities.html to launch a runtime:
+
+To open the specification model, you will need import the project in the workspace of your runtime.
+
+When you opened the specification model, you can open the System Analysis, then the Data folder, then select the Capella Light Metamodel. You can right click on it and select the "Generate Python API from Capella" menu:
```
Comment (mbats) [inline doc/org.eclipse.python4capella.doc/doc/developer/python/GeneratePythonAPI.textile:13]: opened -> open

File: doc/org.eclipse.python4capella.doc/doc/developer/python/GeneratePythonAPI.textile [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+
+This section describes how you can generate Python API for the Capella light metamodel.
+
+The generator is defined in the "ProduceCapellaPythonAPIFromCapellaHandler":https://github.com/labs4capella/python4capella/blob/master/plugins/org.eclipse.python4capella.gen/src/org/eclipse/python4capella/gen/handlers/ProduceCapellaPythonAPIFromCapellaHandler.java Java class. It use the simplified Capella metamodel defined in the "specification folder":https://github.com/labs4capella/python4capella/tree/master/specification of the git repository.
+
+The Capella Light Metamodel is defined using Capella 5.0, you will need this version to generate the Python API.
+
+Start to import the "org.eclipse.python4capella.gen":https://github.com/labs4capella/python4capella/tree/master/plugins/org.eclipse.python4capella.gen project in your Capella 5.0 workspace and "enable developer capabilities":../../user/general/EnableDevelopmentCapabilities.html to launch a runtime:
+
+To open the specification model, you will need import the project in the workspace of your runtime.
```
Comment (mbats) [inline doc/org.eclipse.python4capella.doc/doc/developer/python/GeneratePythonAPI.textile:11]: to import

**Review by mbats — APPROVED** [review]

File: doc/org.eclipse.python4capella.doc/doc/developer/DeveloperManual.textile (hunk 1 above)
Comment (ylussaud) [reply doc/org.eclipse.python4capella.doc/doc/developer/DeveloperManual.textile:12]: done

File: doc/org.eclipse.python4capella.doc/doc/developer/python/GeneratePythonAPI.textile (hunk 2 above)
Comment (ylussaud) [reply doc/org.eclipse.python4capella.doc/doc/developer/python/GeneratePythonAPI.textile:5]: done

File: doc/org.eclipse.python4capella.doc/doc/developer/python/GeneratePythonAPI.textile (hunk 3 above)
Comment (ylussaud) [reply doc/org.eclipse.python4capella.doc/doc/developer/python/GeneratePythonAPI.textile:13]: done

File: doc/org.eclipse.python4capella.doc/doc/developer/python/GeneratePythonAPI.textile (hunk 4 above)
Comment (ylussaud) [reply doc/org.eclipse.python4capella.doc/doc/developer/python/GeneratePythonAPI.textile:11]: done

**Pull request merged**